src/constants/mobile-devices.constants.ts

```diff
@@ -0,0 +1,128 @@
+/**
+ * @description
+ * @regex /\b([A-Z]{3}-)\b/gi
+ * This pattern matches any string of three uppercase letters followed by a hyphen.
+ * @example huaweiDevicesRegex.test("ALP-") // returns true
+ */
+export const huaweiDevicesRegex = /\b([A-Z]{3}-)\b/gi;
```

Comment:
```suggestion
export const huaweiDevicesRegex = /\b([A-Z]{3}-)\b/gi;

export const huaweiDevicesRegex =  /\b(ALP-|AMN-|ANA-|ANE-|ANG-|AQM-|ARS-|ART-|ATU-|BAC-|BLA-|BRQ-|CAG-|CAM-|CAN-|CAZ-|CDL-|CDY-|CLT-|CRO-|CUN-|DIG-|DRA-|DUA-|DUB-|DVC-|ELE-|ELS-|EML-|EVA-|EVR-|FIG-|FLA-|FRL-|GLK-|HMA-|HW-|HWI-|INE-|JAT-|JEF-|JER-|JKM-|JNY-|JSC-|LDN-|LIO-|LON-|LUA-|LYA-|LYO-|MAR-|MED-|MHA-|MLA-|MRD-|MYA-|NCE-|NEO-|NOH-|NOP-|OCE-|PAR-|PIC-|POT-|PPA-|PRA-|RNE-|SEA-|SLA-|SNE-|SPN-|STK-|TAH-|TAS-|TET-|TRT-|VCE-|VIE-|VKY-|VNS-|VOG-|VTR-|WAS-|WKG-|WLZ-|JAD-|MLD-|RTE-|NAM-|NEN-|BAL-|JLN-|YAL-|MGA-|FGD-|XYAO-|BON-|ALN-|ALT-|BRA-|DBY2-|STG-|MAO-|LEM-|GOA-|FOA-|MNA-|LNA-)\b/gi;
```

and then inside `validateHuaweiCodes`:
`const validateHuaweiCodes = (inputString: string) => return huaweiDevicesRegex.test(inputString)`

I think it should work fine